### ISSUE TITLE
A11y_Model Builder_Forecasting_Data_Focus Visible: Keyboard focus is not visible for the graph which is present in the "Data" tab.

### ISSUE THREAD

**Issue by Samhitha66** [dotnet/machinelearning-modelbuilder#2673]
### GitHubTags

#WCAG2.4.7; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Focus Visible; #Win32; #Benchmark; #Keyboard; #A11yAuto; #Tabstops;

### Environment Details

Application Name: Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.1
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621

### Repro steps

1. Launch the Visual Studio with at least one of the .NET workload.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu(Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. Tab till Scenario tab.
7. Select Scenario screen will appear.
8. Tab till "Forecasting" and press enter.
9. Setup environemnt screen will appear, Tab till Local CPU tile and select it.
10. Tab till Next button and press enter.
11. Data screen will appear.
12. Tab till "Graph" radio button after filling all data and press enter.
13. Verify whether the keyboard focus is visible on the graph section or not.

### Actual Result

Keyboard focus is not visible for the graph which is present in the "Data" tab.

Issue observed throughout the application.

### Expected Result

Keyboard focus should be visible for the graph which is present in the "Data" tab.

### User Impact

User will not be able get the information regarding where exactly the keyboard focus present in the page and the information related to that.

### Attachment
![Bug_Focus Visible](https://github.com/dotnet/machinelearning-modelbuilder/assets/99243079/e53ee71d-0ac0-4ce9-bb44-6492b6b0ff37)
![image](https://github.com/dotnet/machinelearning-modelbuilder/assets/99243079/42bdf462-b17c-407e-9cfb-a2ef2be8ad5f)

**Comment by zewditu** [comment]
@Samhitha66  The fuze is also is not on the table preview , are you expecting it should be on the table preview too and  the narrator should announce something ?

**Comment by Samhitha66** [comment]
@zewditu 

1. If we activate table radio button then the keyboard focus order is proper and it is not getting lost.
    It will go to graph radio button and then next button.

2. If we activate Graph radio button then the keyboard focus order is not proper and it is getting lost.
    If we press tab from graph radio button the focus is not visible anywhere and again if we press tab then the focus is going to 
    next button.

#Regressed:06-15-23;

**Comment by zewditu** [comment]
The graph is not an interactive element hence we do not need to set focus on the graph, hence the fix should be after the radio button the focus should go to the next button instead of lost.

**Comment by Samhitha66** [comment]
@zewditu 
If the graph is not focusable then yes the next focus should go to next button.
If this issue is fixed please let me know in which version it is fixed we will check and act accordingly.
Thanks!
#Regressed:06-30-23;

**Comment by zewditu** [comment]
@Samhitha66   this  can be validated by a private build you can download from [here](https://microsoft.sharepoint.com/sites/PrivateBuildforAccessibility/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FPrivateBuildforAccessibility%2FShared%20Documents%2FMicrosoft%2EML%2EModelBuilder%20%2821%29%2Evsix&parent=%2Fsites%2FPrivateBuildforAccessibility%2FShared%20Documents
).

**Comment by Samhitha66** [comment]
@zewditu
We don't have access for the link which you provided in the above comment.
Please provide us the access. So that we will verify this bug.
#Regressed:07-13-23; #Pendinginput;

![image](https://github.com/dotnet/machinelearning-modelbuilder/assets/99243079/178796a2-2aa6-4d3c-8fa2-6ff4e96d551c)

**Comment by Samhitha66** [comment]
@zewditu 
Regressed the issue in the below environment
Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview
Version 17.8.0 Preview 1.0 [33912.440.main]
But graph is not at all loading. But now focus is not going there. But we need to cross check after getting graph aswell.
Please provide your input here.
#Regressed:07-14-23;
![image](https://github.com/dotnet/machinelearning-modelbuilder/assets/99243079/a4555bad-464e-49be-9a18-9f45f962618d)

**Comment by zewditu** [comment]
This issue is because of the unstable behavior of Daytona control,  if you validate the focus is not lost I think it is enough to close this issue.

**Comment by Samhitha66** [comment]
@zewditu
Regressed the issue in the below environment
Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview
Version 17.8.0 Preview 1.0 [33912.440.main]
The issue is fixed. Hence closing the bug.
Please find the attachment for the reference.
#Regressed:07-17-23; #Closed;
![Bug_Focus Graph](https://github.com/dotnet/machinelearning-modelbuilder/assets/99243079/82e5d566-eb7c-48e7-ab9c-c3a153f56969)
